sql-statements/sql-statement-drop-stats.md

```diff
@@ -12,10 +12,10 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-drop-stats/']
 
 ```ebnf+diagram
 DropStatsStmt ::=
-    'DROP' 'STATS' TableName
+    "DROP" "STATS" TableNameList 
 
-TableName ::=
-    Identifier ('.' Identifier)?
```

Comment:
Since TableName is used below, why not keep it?

Reply:
I found most `TableName` is kept, while some of them (e.g. in the `sql-statement-drop-table.md`) are removed. I'm not sure which one is expected. @TomShawn Could you give more suggestion?

Reply:
I think developers have more say on this.

Reply:
OK. I'll keep it :smile: